docs/modules/general/preprocessing.mdx

```diff
@@ -0,0 +1,180 @@
+---
+id: preprocessing
+title: Includes & Preprocessing
+---
+
+## Includes
+
+Include statements allow global XML files to be loaded and reused across different maps, thereby standardizing elements across maps.
+
+Using the include element, an XML file can be split into multiple files and then included in the main one.
+This allows you to keep the main file more organized by including sections such as kits, actions, and more.
+
+```xml
+<map>
+    ...
+    <include id="include-name-goes-here"/>
+    ...
+</map>
+```
+
+### Example
+
+Below is a sample include file that can be used for Blitz maps:
+
+```xml title="/server/includes/blitz.xml"
+<!-- The location for include files is defined in /plugins/PGM/config.yml -->
+<map>
+    <blitz>
+        <lives>5</lives>
+    </blitz>
+</map>
+```
+
+The include file is automatically given an ID based on its file name, which in this case is `blitz`.
+Then, it can be added to the main `map.xml`. Multiple include statements can be used per map.
+
+```xml title="/server/maps/map_name/map.xml"
+<map>
+...
+<!-- All maps with this include statement will give the player 5 lives -->
+<include id="blitz"/>
+...
+</map>
+```
+
+## Map Variants
+
+Mapmakers may choose to build different versions of their maps that are tailored for events, such as tournaments. 
```

Comment:
A better example than tournaments is seasonal maps (halloween/christmas editions), i'd use that